src/peripheral/nvic.rs

```diff
@@ -33,9 +33,16 @@ pub struct RegisterBlock {
     /// Interrupt Active Bit (not present on Cortex-M0 variants)
     #[cfg(not(armv6m))]
     pub iabr: [RO<u32>; 16],
-    #[cfg(armv6m)]
+    #[cfg(any(armv6m, armv8m))]
```

Comment:
This change doesn't seem right to me - armv8m still has IABR right? And in any event the matching cfg above isn't changed, so armv8 would end up with an extra 16 byte offset here.